exchanges/coinut/coinut.go

```diff
@@ -380,50 +380,27 @@ func (c *COINUT) calculateTradingFee(base, quote currency.Code, purchasePrice, a
 }
 
 func getInternationalBankWithdrawalFee(c currency.Code, amount float64) float64 {
-	var fee float64
-
-	switch c {
+	switch c.Upper() {
 	case currency.USD:
-		if amount*0.001 < 10 {
-			fee = 10
-		} else {
-			fee = amount * 0.001
-		}
+		return max(amount*0.001, 10.0)
 	case currency.CAD:
-		if amount*0.005 < 10 {
-			fee = 2
-		} else {
-			fee = amount * 0.005
-		}
+		return max(amount*0.005, 2.0)
 	case currency.SGD:
-		if amount*0.001 < 10 {
-			fee = 10
-		} else {
-			fee = amount * 0.001
-		}
+		return max(amount*0.001, 10.0)
```

Comment:
🚧 I believe this fee is [incorrect](https://coinut.zendesk.com/hc/en-us/articles/19088967379609-4c-XSGD-Withdrawals-via-Bank)
```suggestion
		return 2.0
```